README.md

```diff
@@ -363,15 +363,29 @@ user_profile = UserProfile({
   'customAttr': 'custom value'
 })
 
-print(user_profile.custom_attr)
+print(user_profile.customAttr)
 
 # Creating an empty object and using variables
 user_profile = models.UserProfile()
 user_profile.first_name = 'John'
 user_profile.last_name = 'Doe'
 user_profile.email = 'John.Doe@okta.com'
 user_profile.login = 'John.Doe@okta.com'
-user_profile.custom_attr = 'custom value'
+user_profile.customAttr = 'custom value'
+```
+
+**NOTE**: Custom Attributes should always keep letter case the same as defined in `Profile` tab on your org UI.
+
+It is possible to use original (lowerCamelCase) case for basic attributes to make your code look more consistent and nicer:
```

Comment:
Change:

**NOTE**: Custom Attributes case must be the same as defined in the `Profile` tab of your organization UI.

To maintain consistent code, you can use camelCase for all default attributes

Reply:
Fixed